spec/account/account-manager.spec.js

```diff
@@ -82,7 +82,8 @@ describe("AccountManager", () => {
                 previousMfa: "098454",
                 password: "3439gajs933098fj3jfj90aj09fj9390a9023"
             }).then((result) => {
-                expect(result).toEqual({accountId: "aeifFeight3ighrFieigheilw5lfiek",
+                expect(result).toEqual({
+                accountId: "aeifFeight3ighrFieigheilw5lfiek",
```

Comment:
Do not pass back the accountId, currentMfa, previousMfa, password.  In fact, don't return anything besides the accountIdHash that they will use for future API calls.  I understand that this is just returning what the mock returns, but the manager shouldn't need to expose this stuff.